internal/integrationtest/daemon/daemon_board_watch_test.go

```diff
@@ -0,0 +1,84 @@
+// This file is part of arduino-cli.
+//
+// Copyright 2022 ARDUINO SA (http://www.arduino.cc/)
+//
+// This software is released under the GNU General Public License version 3,
+// which covers the main part of arduino-cli.
+// The terms of this license can be found at:
+// https://www.gnu.org/licenses/gpl-3.0.en.html
+//
+// You can be released from the requirements of the above licenses by purchasing
+// a commercial license. Buying such a license is mandatory if you want to
+// modify or otherwise use the software for commercial activities involving the
+// Arduino software without disclosing the source code of your own applications.
+// To purchase a commercial license, send an email to license@arduino.cc.
+
+package daemon_test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"testing"
+	"time"
+
+	"github.com/arduino/arduino-cli/rpc/cc/arduino/cli/commands/v1"
+	"github.com/stretchr/testify/require"
+)
+
+func TestArduinoCliDaemon(t *testing.T) {
+	// See: https://github.com/arduino/arduino-cli/pull/1804
+
+	t.SkipNow() // TO BE Removed once the bug is fixed
+
+	env, cli := createEnvForDaemon(t)
+	defer env.CleanUp()
+
+	_ = cli.StartDaemon(false)
+
+	inst := cli.Create()
```

Comment:
```suggestion
	grpcInst := cli.Create()
```
This is more clear and cannot be confused with a cli instance. TODO change the other occurrences accordingly